gocourse04/go.mod

```
@@ -0,0 +1,3 @@
+module main
```

Модулі прийнято ось так називати:
```suggestion
module github.com/SerhiiSokolov/go_course/gocourse04
```

Тобто, повний шлях проекту і папка.

В наступні домашці, модуль буде називатись:
```go
module github.com/SerhiiSokolov/go_course/gocourse05
```